cli/borges/consumer.go

```diff
@@ -53,9 +56,24 @@ func (c *consumerCmd) Execute(args []string) error {
 	wp.SetWorkerCount(c.WorkersCount)
 
 	ac := borges.NewConsumer(q, wp)
-	ac.Start()
 
-	return nil
+	var term = make(chan os.Signal)
```

Comment:
Apparently signal channels must be buffered: https://golang.org/pkg/os/signal/#example_Notify

Reply:
Not in our case, we call `signal.Notify` after setting up the goroutine that listens on the channel.